packages/react-error-overlay/webpack.config.js

```diff
@@ -0,0 +1,27 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: 'cheap-module-source-map',
+  entry: './src/iframeScript.js',
```

Comment:
Should we bundle polyfills? 🤔
or we could document it and leave it up to the person importing it ...

Reply:
Yeah, I think we should. Polyfills on parent context don't seem to have an effect on the iframe context. We need polyfills to run in some browsers. (see https://github.com/facebookincubator/create-react-app/issues/3034). I'll update.